server/routes/visit/visitEventsTimelineBuilder.test.ts

```diff
@@ -0,0 +1,303 @@
+        datetime: { timestamp: '2022-01-01T09:00:00', type: 'datetime' },
+        byline: { text: 'User One' },
+        attributes: { 'data-test': 'timeline-entry-0' },
+      },
+    ]
+
+    const timeline = visitEventsTimelineBuilder(params)
+
+    expect(timeline).toStrictEqual(expectedTimeline)
+  })
+
+  it('should return a timeline with an event for "Booked" - GOV.UK Booking', () => {
+    params.events = [
+      {
+        type: 'BOOKED_VISIT',
+        applicationMethodType: 'WEBSITE',
+        actionedByFullName: '',
+        userType: 'PUBLIC',
+        createTimestamp: '2022-01-01T09:00:00',
+      },
+    ]
+
+    const expectedTimeline: MojTimelineItem[] = [
+      {
+        label: { text: 'Booked' },
+        text: 'Method: GOV.UK booking',
+        datetime: { timestamp: '2022-01-01T09:00:00', type: 'datetime' },
+        attributes: { 'data-test': 'timeline-entry-0' },
+      },
+    ]
+
+    const timeline = visitEventsTimelineBuilder(params)
+
+    expect(timeline).toStrictEqual(expectedTimeline)
+  })
+
+  it('should return a timeline with an event for "Updated" - prisoner request', () => {
+    params.events = [
+      {
+        type: 'UPDATED_VISIT',
+        applicationMethodType: 'BY_PRISONER',
+        actionedByFullName: 'User One',
+        userType: 'STAFF',
+        createTimestamp: '2022-01-01T09:00:00',
+      },
+    ]
+
+    const expectedTimeline: MojTimelineItem[] = [
+      {
+        label: { text: 'Updated' },
+        text: 'Method: Prisoner request',
+        datetime: { timestamp: '2022-01-01T09:00:00', type: 'datetime' },
+        byline: { text: 'User One' },
+        attributes: { 'data-test': 'timeline-entry-0' },
+      },
+    ]
+
+    const timeline = visitEventsTimelineBuilder(params)
+
+    expect(timeline).toStrictEqual(expectedTimeline)
+  })
+
+  it('should return a timeline with an event for "Updated" - GOV.UK Booking', () => {
+    params.events = [
+      {
+        type: 'UPDATED_VISIT',
+        applicationMethodType: 'WEBSITE',
+        actionedByFullName: '',
+        userType: 'PUBLIC',
+        createTimestamp: '2022-01-01T09:00:00',
+      },
+    ]
+
+    const expectedTimeline: MojTimelineItem[] = [
+      {
+        label: { text: 'Updated' },
+        text: 'Method: GOV.UK booking',
+        datetime: { timestamp: '2022-01-01T09:00:00', type: 'datetime' },
+        attributes: { 'data-test': 'timeline-entry-0' },
+      },
+    ]
+
+    const timeline = visitEventsTimelineBuilder(params)
+
+    expect(timeline).toStrictEqual(expectedTimeline)
+  })
+
+  it('should return a timeline with an event for "Cancelled" - Reason: /free text/', () => {
+    params.visitNotes = [
+      {
+        type: 'VISIT_OUTCOMES',
+        text: 'Cancelled due to illness',
+      },
+    ]
+
+    params.events = [
+      {
+        type: 'CANCELLED_VISIT',
+        applicationMethodType: 'WEBSITE',
+        actionedByFullName: 'User One',
+        userType: 'STAFF',
```

Comment:
This was previously `userType: 'PUBLIC'` but I have changed it to `STAFF`. This is the only change made to the existing tests (other than moving them!)

@hutcheonb-moj I am assuming this was incorrect in the original test data because a free text cancellation reason is only captured when a visit is cancelled in the staff service. Does that sound right?